app/models/report.rb

```diff
@@ -135,6 +135,11 @@ def build_report
     sys = SystemSettings.current
     primary_range = sys.primary_age_range
     age_ranges = sys.age_ranges[primary_range]
+		
+		if self.data.present?
+      self.data = { values: self.data };
```

Comment:
I'm concerned that this short-circuits the basic way reports work.  Here it seems the report data is being fetched from the DB when the report is created and saved on the report record, then just the same report data re-displayed over and over each time the user clicks to view the report.  Normally, a report, as saved in the db, is just info on how to retrieve the report data from solr.  In that scenario, fresh report data is fetched each time a user clicks on a report to view the report.